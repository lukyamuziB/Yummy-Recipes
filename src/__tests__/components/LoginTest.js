import React from 'react';
import Enzyme, { mount, shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import { BrowserRouter, MemoryRouter } from 'react-router-dom';
import thunk from 'redux-thunk';
import { Provider } from 'react-redux';
import configureMockStore from 'redux-mock-store';
import Login from '../../components/loginUser';


describe('<Login/>', () => {
  const component = shallow(
    <BrowserRouter>
      <Login />
    </BrowserRouter>);
  it('should render without crashing', () => {
    expect(component).toHaveLength(1);
  });
  it('should render without crashing', () => {
    expect(component).toHaveLength(1);
  });
});

