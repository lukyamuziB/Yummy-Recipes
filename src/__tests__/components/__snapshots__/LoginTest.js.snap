// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Login/> renders fully 1`] = `
<div
  id="Login"
>
  <div
    className="landing-container"
  >
    <div
      className="in-container"
    >
      <div
        className="inner"
      >
        <div
          className="container"
        >
          <div
            className="container"
          >
            <h5>
               Login Here
            </h5>
             
            <br />
            <form
              onSubmit={[Function]}
            >
              <div
                className="form-group %{touched && error ? 'has-dangetr':''}"
              >
                <lable>
                   
                  Username
                   
                </lable>
                <input
                  className="form-control"
                  name="username"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onDragStart={[Function]}
                  onDrop={[Function]}
                  onFocus={[Function]}
                  type="text"
                  value=""
                />
                
              </div>
              <div
                className="form-group %{touched && error ? 'has-dangetr':''}"
              >
                <lable>
                   
                  Password
                   
                </lable>
                <input
                  className="form-control"
                  name="password"
                  onBlur={[Function]}
                  onChange={[Function]}
                  onDragStart={[Function]}
                  onDrop={[Function]}
                  onFocus={[Function]}
                  type="password"
                  value=""
                />
                
              </div>
              <button
                className="btn btn-primary"
                type="submit"
              >
                login
              </button>
               
              <br />
              <br />
               Don't have an account yet?  
              <a
                href="/register"
                onClick={[Function]}
              >
                 Register here 
              </a>
            </form>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
